aten/src/ATen/TensorIterator.cpp

```diff
@@ -797,6 +797,9 @@ bool TensorIteratorBase::is_contiguous() const {
 bool TensorIteratorBase::is_scalar(int arg) const {
```

Comment:
Maybe we should replace the whole function with something similar to `Tensor.item()`, which simply checks for `numel`.
https://github.com/pytorch/pytorch/blob/9bda1e874c4048596966ab7e7b8998c4379e09f8/aten/src/ATen/native/Scalar.cpp#L17-L29